dash/__init__.py

```diff
@@ -53,3 +53,37 @@ def _jupyter_nbextension_paths():
             "require": "dash/main",
         }
     ]
+
+
+__all__ = [
```

Comment:
why such a big list - was `__all__` not defined at all before?

Reply:
With the addition of `py.typed` file, it is now expected to have a `__all__` attribute, otherwise it shows an error on every imports. The list contains everything that is "exported" here.

Reply:
:+1: thank you